docker/README.md

```diff
@@ -156,8 +156,7 @@ There are three modes for plugins
 1. Plugins run in the `fiftyone-app` deployment
     - To enable this mode, use the file
       [./compose.plugins.yaml](./compose.plugins.yaml)
-      instead of
-      [./compose.yaml](./compose.yaml)
+      instead of [./compose.yaml](./compose.yaml)
```

Comment:
Can't fix this line in README.md because it's coming from the helm chart description:
`FiftyOne Teams is the enterprise version of the open source [FiftyOne](https://github.com/voxel51/fiftyone) project.`

So all attempts to regex check this standard will fail forever?

Reply:
> ...and the markdown linter didn't call it out, so by what standard should links be on their own lines?

I don't have a link to it. I saw it once and just adopted that convention. Unfortunately linters and static code analyzers may not always catch every case.